tracking_service/src/main/java/com/tel_ran/tracking_service/entity/CustomerEntity.java

```diff
@@ -23,4 +23,14 @@
     @OneToMany
     List<ShipmentEntity> shipments;
 
+    public CustomerEntity(String firstName, String lastName) {
+        this.firstName=firstName;
+        this.lastName=lastName;
+    }
+
+    public CustomerEntity(Long customer_id,String firstName, String lastName) {
+        this.customer_id = customer_id;
```

Comment:
We never use underscore (snake case notation) in the field names in Java. customer_id -> customerId.